packages/react/src/Pagination/Pagination.module.css

```
@@ -7,57 +7,27 @@
 .Pagination__inner {
   display: inline-flex;
   align-items: center;
+  gap: var(--base-size-8);
 }
 
 .Pagination__item {
-  min-width: var(--base-size-36);
+  min-width: var(--base-size-32);
+  height: var(--base-size-32);
   text-align: center;
   white-space: nowrap;
   vertical-align: middle;
   cursor: pointer;
   user-select: none;
   text-decoration: none;
   padding: var(--base-size-4) var(--base-size-8);
-  margin-right: var(--base-size-8);
   border: var(--brand-borderWidth-thick) solid transparent;
   border-radius: var(--brand-borderRadius-medium);
   transition: border-color var(--brand-animation-duration-fast) var(--brand-animation-easing-default);
 }
 
-.Pagination__item--hidden,
-.Pagination__item--hidden-narrow,
-.Pagination__item--hidden-regular,
-.Pagination__item--hidden-wide {
-  display: none;
-}
-
-.Pagination__item--visible,
-.Pagination__item--visible-narrow {
-  display: block;
-}
-
-@media screen and (min-width: 48rem) {
-  .Pagination__item--hidden-regular {
-    display: none;
-  }
-
-  .Pagination__item--visible-regular {
-    display: block;
-  }
-}
-
-@media screen and (min-width: 80rem) {
-  .Pagination__item--hidden-wide {
-    display: none;
-  }
-
-  .Pagination__item--visible-wide {
-    display: block;
-  }
-}
-
-.Pagination__item:last-child {
-  margin-right: 0;
+.Pagination__item * {
+  font-size: var(--brand-text-size-100);
+  line-height: 20px;
```

As we try not to use px values directly...

```suggestion
  line-height: var(--base-size-20);
```

---

Good spot, thanks!